lib/cocoapods/installer.rb

```diff
@@ -422,11 +422,9 @@ def run_plugins_pre_install_hooks
     # @return [void]
     #
     def perform_post_install_actions
-      unlock_pod_sources
       run_plugins_post_install_hooks
       warn_for_deprecations
       warn_for_installed_script_phases
-      lock_pod_sources
```

Comment:
Don't we still want the sources to end up locked after installation? Or is there another mechanism for making that happen?

Reply:
oof... good catch. at the very least, we can skip unlocking ? So 1/2 as much benefit?